.devcontainer/devcontainer.json

```diff
@@ -5,8 +5,7 @@
 		"ghcr.io/devcontainers/features/azure-cli:1": {
 			"version": "latest"
 		},
-		"ghcr.io/azure/azure-dev/azd:latest": {},
-		"rchaganti/vsc-devcontainer-features/azurebicep": {}
```

Comment:
I guess you don't need this anymore, but is there any other reason to remove it?

Reply:
The extension installs the binary now